docs(firestore-bigquery-export): delete query (#463)
firebase/extensions: firestore-bigquery-export/guides/EXAMPLE_QUERIES.md

all `DISTINCT` documents to eliminate the redundant rows introduced by the
cartesian product of `CROSS JOIN`.

```sql
SELECT DISTINCT document_name, favorite_numbers_member
FROM ${param:PROJECT_ID}.users.users_schema_user_full_schema_latest
WHERE favorite_numbers_index = 0
```

### Remove stale data from your changelog table

> If you wish to clean up the data from your `changelog` table, use the following `DELETE` query to provide
> a range of time against the `timestamp` column, and delete the requisite rows.

Reviewer: ```suggestion
If you want to clean up data from your `changelog` table, use the following
`DELETE` query to delete the rows that fall within a range of timestamps.
```

```sql

> /* the first WHERE query is the start of the time range, the second WHERE query is the end of the time range. The below deletes every row in a 24 hour time frame on the 4th September, 2020*/

Reviewer: ```suggestion
/* The first WHERE clause is the start of the time range, and the second WHERE
   clause is the end of the time range. The query below deletes every row in a
   24-hour time frame on the 4th September, 2020. */
```
